### PR TITLE
Update RELEASING file with more information

### PR DESCRIPTION
**Pull request by captainsafia**
These updates:

- Establish a weekly release cadence for packages
- Establish a biweekly release cadence for the desktop app
- Clarify some instructions in the releasing doc for desktop releases

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -1,21 +1,36 @@
...
+```
 
-In order to build a signed copy with working auto-update, you will need to join the Apple developer program and get a certificate. The [Electron docs have a document on submitting your app to the app store](https://github.com/electron/electron/blob/master/docs/tutorial/mac-app-store-submission-guide.md), you only have to get through to the certificate step.
+Note that `setx` will apply the environment variable in subsequence Command Prompt sessions.
 
-## Release Process
 **Note: Make sure that your remote origin is set as `git@github.com:nteract/nteract.git`.**
 
-1.  Make sure the release is working by running `yarn dist` and testing the built app inside the `./applications/desktop/dist/` folder. You can build for all platforms using `yarn dist:all`.
+1.  Make sure the release is working by running `yarn dist` and testing the built app inside the `./applications/desktop/dist/` folder. You can build for all platforms using `yarn dist:all`. Note that you can only build for all platforms on macOS. macOS builds cannot be built on Windows and Linux.
```
Comment (MSeal) [inline RELEASING.md:33]: > macOS builds cannot be built on Windows and Linux.

:(

Comment (MSeal) [reply RELEASING.md:33]: But a good doc addition as I could see this leading someone astray if they didn't know.

**Comment by willingc** [comment]
A bit off topic: do we build binaries nightly for nteract desktop? Might make sense to do so and keep somewhere.

**Comment by BenRussert** [comment]
> do we build binaries nightly 

Not that I know of, but I think nightlies would be a good idea @willingc.

**Comment by captainsafia** [comment]
We don't have nightly builds at the moment although we could configure them in a CircleCI build. I made an issue for it in https://github.com/nteract/nteract/issues/4328.

**Review by stormpython — APPROVED** [review]

**Pull request merged**